deps/spidershim/src/conversions.h

```diff
@@ -40,6 +40,10 @@ static inline Value* GetV8Value(JS::Value* val) {
   return reinterpret_cast<Value*>(val);
 }
 
+static inline Value* GetV8Value(JS::MutableHandleValue val) {
```

Comment:
On Thu, May 12, 2016 at 03:32:20PM -0700, Ehsan Akhgari wrote:
> In deps/spidershim/src/conversions.h:
> 
> > @@ -40,6 +40,10 @@ static inline Value* GetV8Value(JS::Value* val) {
> >    return reinterpret_cast<Value*>(val);
> >  }
> >
> > +static inline Value* GetV8Value(JS::MutableHandleValue val) {
> 
> Hmm, passing in a MutableHandleValue here looks wrong, since that type is
> intended only for out arguments... As far as I can tell, GCVector::operator[]
> returns a `const JS::Value&' so I'm not sure why you need this overload.

If you just read GCVector you'd reasonably think that, but haha noap.
Turns out PersistentRooted<T> inherits from PersistentRootedBase<T>
which is specialized for GCVector to inherit from
MutableGCVectorOperations<...>, which defines an operator[] returning
MutableHandle.  So, maybe we could do something else probably involving
some const casting, but just deleting the overload won't work.